spikeinterface/core/basesnippets.py

```diff
@@ -0,0 +1,530 @@
+from typing import List, Union
+from pathlib import Path
+from .base import BaseExtractor, BaseSegment
+import numpy as np
+from warnings import warn
+from probeinterface import Probe, ProbeGroup, write_probeinterface, read_probeinterface, select_axes
+
+#snippets segments? 
+
+class BaseSnippets(BaseExtractor):
```

Comment:
@ferchaure I'm thinking that this could inherit from the `BaseRecording` and override a few functions as not implemented (e.g. `get_traces()`).
 This would allow to have all the basic functions (`get_sampling_frequency()`) plus all the probe related ones. What do you think? @samuelgarcia any input?

Reply:
yes, you can push. I´m not sure how it works but i will be careful